src/components/Home/Home.test.tsx

```diff
@@ -13,17 +13,6 @@ describe('Home component tests', () => {
     expect(passwordLengthSpan).toBeInTheDocument();
   });
 
-  test('It updates word length counter', () => {
```

Comment:
This test should still exist. Think of it as: The Home component needs to test it's own behaviour and the behaviour of its child components